syntax = "proto3";

package fbsrankings.messages.query;

message TeamCountBySeasonResult {
    string query_id = 1;
    string season_id = 2;
    int32 count = 3;
}

message TeamCountBySeasonQuery {
    string query_id = 1;
    string season_id = 2;
}
